src/assets/scripts/client/ui/UiController.js

```diff
@@ -387,6 +393,8 @@ class UiController {
      * @private
      */
     _onClickCloseAirportDialog() {
+        EventTracker.trackEvent(TRACKABLE_EVENT.AIRPORTS, 'airport-switcher', 'close');
+
         this.$airportDialog.removeClass(SELECTORS.CLASSNAMES.OPEN);
```

Comment:
The empty line is probably not needed, since both of these are method calls. If you think it should be this way, then there are others to change to be like this.